charts/cluster-registrator/templates/hooks/pre-delete-hook.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: pre-delete-job
+  namespace: nirmata
+  labels:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: hook-succeeded
+spec:
+  template:
```

Comment:
Also, we need to give the cluster registrator service account as the SA running the hook. Just check in kyverno-operator pre-delete hook.